index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Laika Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
looks like the path to the CSS file is wrong,
try looking to the project folder to verify please